### DISCUSSION TITLE
SDS not working for GB English

### DISCUSSION THREAD

**Discussion by vdaskalov123** [Mr-MIBonk/M.I.B._More-Incredible-Bash#583] [General]
Hi all, 

I did EU region conversion on my Audi Q7 2018 (4M)  MHI2Q with M.I.B V3.6.0  I have also patched it and updated with latest maps. 
The issue that I have is that SDS is not working for GB English. For the other languages is working just fine. 
I have tried already to re-flash the Speech modules , re-patched and restored all to factory with no luck. 
Any help on this is appreciated.

**Comment by vdaskalov123** [comment]
MIB had a wrong language dataset. I have added new setlang script with the correct dataset and this fixed the issue. If anyone else run into same issue please ping me  vdaskalov@gmail.com

**Comment by neoxic** [comment]
The fix is here: https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/issues/827#issuecomment-3966690898

**Reply by chiritalaur-pixel** [reply]
Thanks, it worked!

**Reply by chiritalaur-pixel** [reply]
So, SDS is working but now I have 1588 - invalid data record + 1598 - Data Set Implausible . VIM workaround doesn't work. Any idea how to get rid of these 2 errors? Thank you
